packages/jest-phabricator-coverage/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+function formatResults(testResult, coverageMap) {
```

Comment:
Can we potentially reconcile this with https://github.com/cpojer/jest/blob/master/packages/jest-cli/src/lib/formatTestResults.js#L22 which already exists?